projects/1.18/assets/ecologics/ecologics/lang/zh_cn.json

```
@@ -0,0 +1,226 @@
+  "block.ecologics.walnut_hedge": "核桃树叶篱",
+  "block.ecologics.walnut_chest": "核桃木箱子",
+  "block.ecologics.walnut_trapped_chest": "核桃木陷阱箱",
+  "block.ecologics.walnut_leaf_carpet": "核桃树叶地毯",
+  "block.ecologics.vertical_walnut_planks": "竖纹核桃木板",
+  "block.ecologics.seashell_tile_vertical_slab": "贝壳瓦竖直台阶",
+  "block.ecologics.ice_brick_vertical_slab": "冰砖竖直台阶",
+  "block.ecologics.snow_brick_vertical_slab": "雪砖竖直台阶",
+  "block.ecologics.azalea_beam": "杜鹃木支撑柱",
+  "block.ecologics.azalea_palisade": "杜鹃木实心栅栏",
+  "block.ecologics.azalea_seat": "杜鹃木板凳",
+  "block.ecologics.azalea_support": "杜鹃木支架",
+  "block.ecologics.coconut_beam": "椰木支撑柱",
+  "block.ecologics.coconut_palisade": "椰木实心栅栏",
+  "block.ecologics.coconut_seat": "椰木板凳",
+  "block.ecologics.coconut_support": "椰木支架",
+  "block.ecologics.flowering_azalea_beam": "杜鹃花木支撑柱",
+  "block.ecologics.flowering_azalea_palisade": "杜鹃花木实心栅栏",
+  "block.ecologics.flowering_azalea_seat": "杜鹃花木板凳",
+  "block.ecologics.flowering_azalea_support": "杜鹃花木支架",
+  "block.ecologics.walnut_beam": "核桃木支撑柱",
+  "block.ecologics.walnut_palisade": "核桃木实心栅栏",
+  "block.ecologics.walnut_seat": "核桃木板凳",
+  "block.ecologics.walnut_support": "核桃木支架",
+  "block.ecologics.azalea_log_bridge_middle": "杜鹃木桥",
+  "block.ecologics.flowering_azalea_log_bridge_middle": "杜鹃花木桥",
+  "block.ecologics.coconut_log_bridge_middle": "椰木桥",
+  "block.ecologics.walnut_log_bridge_middle": "核桃木桥",
+  "block.ecologics.rope_azalea_bridge": "杜鹃木索桥",
+  "block.ecologics.rope_flowering_azalea_bridge": "杜鹃花木索桥",
+  "block.ecologics.rope_coconut_bridge": "椰木索桥",
+  "block.ecologics.rope_walnut_bridge": "核桃木索桥",
+  "block.ecologics.azalea_bridge_pier": "杜鹃木桥梁支架",
+  "block.ecologics.flowering_azalea_bridge_pier": "杜鹃花木桥梁支架",
+  "block.ecologics.coconut_bridge_pier": "椰木桥梁支架",
+  "block.ecologics.walnut_bridge_pier": "核桃木桥梁支架",
+  "block.ecologics.azalea_log_bridge_stair": "杜鹃木桥梯",
+  "block.ecologics.flowering_azalea_log_bridge_stair": "杜鹃花木桥梯",
+  "block.ecologics.coconut_log_bridge_stair": "椰木桥梯",
+  "block.ecologics.walnut_log_bridge_stair": "核桃木桥梯",
+  "block.ecologics.azalea_rope_bridge_stair": "杜鹃木索桥梯",
+  "block.ecologics.flowering_azalea_rope_bridge_stair": "杜鹃花木索桥梯",
+  "block.ecologics.coconut_rope_bridge_stair": "椰木索桥梯",
+  "block.ecologics.walnut_rope_bridge_stair": "核桃木索桥梯",
+  "block.ecologics.azalea_rail_bridge": "杜鹃木铁轨桥",
+  "block.ecologics.flowering_azalea_rail_bridge": "杜鹃花木铁轨桥",
+  "block.ecologics.coconut_rail_bridge": "椰木铁轨桥",
+  "block.ecologics.walnut_rail_bridge": "核桃木铁轨桥",
+  "death.attack.coconut": "%1$s被掉落的椰子砸到了",
+  "death.attack.coconut.player": "%1$s在与%2$s战斗时被掉落的椰子砸到了",
+  "ecologics.subtitles.block.coconut.smash": "椰子：坠落",
+  "ecologics.subtitles.block.thin_ice.crack": "薄冰：破裂",
+  "ecologics.subtitles.entity.coconut_crab.ambient": "椰子蟹：嘶嘶声",
+  "ecologics.subtitles.entity.coconut_crab.death": "椰子蟹：死亡",
+  "ecologics.subtitles.entity.coconut_crab.hurt": "椰子蟹：受伤",
+  "ecologics.subtitles.entity.penguin.ambient": "企鹅：嘎嘎声",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "ecologics.subtitles.entity.coconut_crab.ambient": "椰子蟹：嘶嘶",
  "ecologics.subtitles.entity.coconut_crab.death": "椰子蟹：死亡",
  "ecologics.subtitles.entity.coconut_crab.hurt": "椰子蟹：受伤",
  "ecologics.subtitles.entity.penguin.ambient": "企鹅：嘎嘎",
```

---

当时对照原版字幕看的

---

这里有反例，原版也混乱
https://github.com/CFPAOrg/Glossary/blob/104816b3d8ff6520dccdba9ab5a8a51b68c31546/Minecraft%20Vanilla%201.18.2.csv#L4965